com.unity.render-pipelines.high-definition/Runtime/Lighting/Reflection/HDProbe.cs

```diff
@@ -128,6 +126,18 @@ float aspect
         // Array of names that will be used in the Render Loop to name the probes in debug
         internal string[] probeName = new string[6];
 
+        float m_ProbeExposureValue = 1.0f;
```

Comment:
Moved the exposure async request logic to the camera. Instead the probe just caches the exposure value.